Packages/App/Sources/CommonLibrary/IAP/IAPManager+Verify.swift

```diff
@@ -37,7 +37,7 @@ extension IAPManager {
     public func verify(_ features: Set<AppFeature>) throws {
 #if os(tvOS)
         guard isEligible(for: .appleTV) else {
-            throw AppError.ineligibleProfile([.appleTV])
+            throw AppError.ineligibleProfile(features.union([.appleTV]))
```

Comment:
Now that the paywall supports multiple purchases, list all required features, not just .appleTV.